java/tools/src/java/org/apache/orc/tools/PrintData.java

```diff
@@ -207,6 +207,7 @@ static void printJsonData(PrintStream printStream,
       TypeDescription schema = reader.getSchema();
       VectorizedRowBatch batch = schema.createRowBatch();
       Integer counter = 0;
+      label:
```

Comment:
Can we change the logic here avoiding labels?
Want to make sure readability here

Reply:
@pgaref I will try to change (I changed the status of this to DRAFT). But, I think that the 1st code is the simplest way to fix this issue.